.travis.yml

```diff
@@ -40,3 +48,31 @@ script:
   - curl -k https://localhost:444
   - curl -k https://localhost:8443
   - curl -k https://localhost:8444
+
+after_success:
+  - curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
+  - chmod +x ./kubectl
+  - sudo mv ./kubectl /usr/local/bin/kubectl
+  - echo -n ${travis} | base64 -d > travis-service-account.json
+  - gcloud auth activate-service-account travis@api-platform-demo-209807.iam.gserviceaccount.com --key-file travis-service-account.json --project=api-platform-demo-209807
+  - gcloud config set compute/zone europe-west3-c
+  - gcloud config set project api-platform-demo-209807
+  - tar -zxvf helm-v2.7.2-linux-amd64.tar.gz
+  - rm -rf helm-v2.7.2-linux-amd64.tar.gz
+  - sudo mv linux-amd64/helm /usr/local/bin/helm
+  - gcloud container clusters get-credentials api-platform-demo --zone europe-west3-c --project ${projectName}
+  - helm init
+  - helm dependencies update ./api/helm/api
+  - docker build --pull -t gcr.io/api-platform-demo-209807/php -t gcr.io/api-platform-demo-209807/php:latest api --target api_platform_php
+  - docker build --pull -t gcr.io/api-platform-demo-209807/nginx -t gcr.io/api-platform-demo-209807/nginx:latest api --target api_platform_nginx
+  - docker build --pull -t gcr.io/api-platform-demo-209807/varnish -t gcr.io/api-platform-demo-209807/varnish:latest api --target api_platform_varnish
+  - gcloud docker -- push gcr.io/api-platform-demo-209807/php:latest
+  - gcloud docker -- push gcr.io/api-platform-demo-209807/nginx:latest
+  - gcloud docker -- push gcr.io/api-platform-demo-209807/varnish:latest
+  - helm install --namespace=${TRAVIS_COMMIT} ./api/helm/api --wait --set php.repository=gcr.io/api-platform-demo-209807/php:latest --set nginx.repository=gcr.io/api-platform-demo-209807/nginx:latest --set secret=${APP_SECRET} --set postgresUser=${DATABASE_USER},postgresPassword="${DATABASE_PASSWORD}",postgresDatabase=${DATABASE_NAME} --set postgresql.persistence.enabled=true --set corsAllowOrigin='*'
+  - kubectl --namespace=${TRAVIS_COMMIT} get pods -l app=api-php -o jsonpath="{.items[0].metadata.name}"
+  - kubectl exec -it $(kubectl --namespace=${TRAVIS_COMMIT} get pods -l app=api-php -o jsonpath="{.items[0].metadata.name}") --namespace=${TRAVIS_COMMIT} -- ash -c 'export APP_ENV=dev && composer install -n && bin/console hautelook:fixtures:load -n && APP_ENV=prod composer --no-dev install --classmap-authoritative && bin/console d:s:u --env=prod'
```

Comment:
Can you add a comment like `Install the dev deps, initialize the DB with the fixtures, and remove the dev deps`